service-setup/README.md

```diff
@@ -0,0 +1,26 @@
+# Quickstart
+To install RuuviCollector as a service and write collected data to an InfluxDB
+database, you can run setup.sh from this directory.
```

Comment:
Since this script requires you to have either cloned or downloaded this repository (as opposed to just grabbing the setup script or this directory), I think it's worth mentioning it ".. you can clone or download this repository and then run setup.sh from this directory" or something.

Perhaps the setup script could even be improved to do this check, I think checking the presence of the sample config in the parent directory is enough since it's very unlikely to be there in any other case except when the repository is cloned or downloaded

Reply:
Updated this and the main README.